xrpl/models/requests/ledger_entry.py

```diff
@@ -206,17 +239,28 @@ def _get_errors(self: LedgerEntry) -> Dict[str, str]:
             for param in [
                 self.index,
                 self.account_root,
-                self.directory,
-                self.offer,
-                self.ripple_state,
                 self.check,
+                self.deposit_preauth,
+                self.directory,
                 self.escrow,
+                self.offer,
                 self.payment_channel,
-                self.deposit_preauth,
+                self.ripple_state,
                 self.ticket,
+                self.xchain_claim_id,
+                self.xchain_create_account_claim_id,
             ]
             if param is not None
         ]
-        if len(query_params) != 1:
+        if (
+            len(query_params) != 1
+            and self.bridge is None
+            and self.bridge_account is None
+        ):
             errors["LedgerEntry"] = "Must choose exactly one data to query"
+
+        if (self.bridge is not None) != (self.bridge_account is not None):
+            # assert that you either have both of these or neither
+            errors["Bridge"] = "Must include both `bridge` and `bridge_account`."
```

Comment:
If this comment is correct, would this be a better error message?
```suggestion
            errors["Bridge"] = "Must include both `bridge` and `bridge_account` or neither."
```

Reply:
The `neither` case isn't really relevant IMO (and would be more confusing), since you're not likely to include one while intending to include neither.